### PR TITLE
feat(notification): add Pushover

### PR DESCRIPTION
**Pull request by zaosoula**
## Changes
- Add the necessary change to allow user to configure Pushover notification channel
IMG_7235

### CODE REVIEW

**Comment by zaosoula** [comment]
cc @peaklabs-dev. 
Still need to complete the checklist but here is the updated version of the PR. I restarted from scratch following the changes in https://github.com/coollabsio/coolify/pull/4525/files#diff-02e04e3fad6de6c4e7dac846540a30327eed2f3f2d1cc1bf927130e9dd7b24f4

**Comment by peaklabs-dev** [comment]
Thank you very much. How do I test it, I have never used Pushover before?

**Comment by zaosoula** [comment]
@peaklabs-dev To test it you, can follow this guide (random link found on google, but the config steps are similar) https://www.cybercom-software.com/help/phonepadadmin/pushover2.htm

Then you can receive notification on your devices

**Comment by peaklabs-dev** [comment]
I will check

File: app/Livewire/Notifications/Pushover.php [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+    #[Validate(['boolean'])]
+    public bool $serverUnreachablePushoverNotifications = true;
+
+
+    public function mount()
+    {
+        try {
+            $this->team = auth()->user()->currentTeam();
+            $this->settings = $this->team->pushoverNotificationSettings();
```
Comment (peaklabs-dev) [inline app/Livewire/Notifications/Pushover.php:67]: This line can not have `()`. it needs to be:
`$this->settings = $this->team->pushoverNotificationSettings;`

Comment (peaklabs-dev) [reply app/Livewire/Notifications/Pushover.php:67]: @zaosoula Found it (at least this should be the issue).

Comment (zaosoula) [reply app/Livewire/Notifications/Pushover.php:67]: Perfect thank you
I will proceed with the testing

Comment (zaosoula) [reply app/Livewire/Notifications/Pushover.php]: It's working
Thanks for the help
IMG_7234

**Comment by peaklabs-dev** [comment]
Thank you. So is it ready to be merged?
Can you please open a Docs PR with an easy to follow step by step guide on how to set it up?

**Comment by peaklabs-dev** [comment]
Also I do not understand what to use for the user I need to use my user key right?
And the token is an API token from an application correct?

**Comment by zaosoula** [comment]
@peaklabs-dev The user and token values got inverted by an error in my code it fixed now.

**Comment by peaklabs-dev** [comment]
Give me a minute to test if it works, If I can get it to work I will do the docs for you.

File: app/Notifications/Dto/PushoverMessage.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    public function __construct(
+        public string $title,
+        public string $message,
+        public array $buttons = [],
+        public string $level = 'info',
+    ) {}
+
+    public function getLevelIcon(): string
+    {
+        return match ($this->level) {
```
Comment (peaklabs-dev) [inline app/Notifications/Dto/PushoverMessage.php:18]: Is it possible to use the same icons that we use for the other notification providers?

Comment (peaklabs-dev) [reply app/Notifications/Dto/PushoverMessage.php:18]: The rest looks good to me so it is almost ready to be merged.

Comment (zaosoula) [reply app/Notifications/Dto/PushoverMessage.php:18]: I'm working on it 👍

Comment (zaosoula) [reply app/Notifications/Dto/PushoverMessage.php:18]:
```php
return match ($this->level) {
            'info' => 'ℹ️',
            'error' => '❌',
            'success' => '✅ ',
            'warning' => '⚠️',
        };
```

Comment (zaosoula) [reply app/Notifications/Dto/PushoverMessage.php:18]: I've updated them to those

**Comment by peaklabs-dev** [comment]
Thank you for the PR and your fast fixing of the issues. I will add a few small cosmetic fixes.

**Pull request merged**